examples/acoustic/fwi_operators.py

```diff
@@ -11,27 +11,44 @@ class ForwardOperator(Operator):
     def __init__(self, model, src, damp, data, time_order=2, spc_order=6,
                  save=False, **kwargs):
         nrec, nt = data.shape
-        dt = model.get_critical_dt()
+        s, h = symbols('s h')
         u = TimeData(name="u", shape=model.get_shape_comp(), time_dim=nt,
-                     time_order=time_order, space_order=spc_order, save=save,
+                     time_order=2, space_order=spc_order, save=save,
```

Comment:
I wouldn't yet, but yes you can. 4th order in time for acoustic is a bit peculiar and only needs second order time derivatives.